test/test_alarm.js

```diff
@@ -310,17 +310,17 @@ describe('ical-generator Alarm', function () {
     describe('interval()', function () {
         it('setter should return this', function () {
             const a = new ICalAlarm(null, new ICalEvent(null, new ICalCalendar()));
-            assert.deepEqual(a, a.interval(null));
-            assert.deepEqual(a, a.interval(60));
+            assert.deepStrictEqual(a, a.interval(null));
+            assert.deepStrictEqual(a, a.interval(60));
         });
 
         it('getter should return value', function () {
             const a = new ICalAlarm(null, new ICalEvent(null, new ICalCalendar()));
-            assert.equal(a.interval(), null);
+            assert.strictEqual(a.interval(), undefined);
```

Comment:
I edited the test to check for undefined since that's the default value of `this._data.interval`